drake/control_simulation/simulation_connector.py

```diff
@@ -0,0 +1,318 @@
+## simulation_connector.py
+#   Description:
+#       This function is meant to provide a connection module to the meshcat simulator for our Kinova arm.
+#       This is built based on infinity_simulation.py
+##
+
+###########
+# Imports #
+###########
+
+from pydrake.all import *
+import numpy as np
+import matplotlib.pyplot as plt
+
+# Start a single meshcat server instance to use for the remainder of this notebook.
+server_args = []
+from meshcat.servers.zmqserver import start_zmq_server_as_subprocess
+proc, zmq_url, web_url = start_zmq_server_as_subprocess(server_args=server_args)
```

Comment:
It seems like this is the main difference between this script and `infinity_simulation.py`. Are there any other changes, because I can't seem to find them.

Reply:
I didn't intended to change this part of the code. I basically created a duplicate for bouncing balls. I wanted to construct interfaces for control modules while I needed to refer to the example. That's why I commented out lines of codes.
I can separate them make it customized specifically for bouncing balls.